python/src/aac/lang/active_context_lifecycle_manager.py

```diff
@@ -22,13 +22,20 @@ def get_active_context(reload_context: bool = False) -> LanguageContext:
 
     if not ACTIVE_CONTEXT or reload_context:
         ACTIVE_CONTEXT = LanguageContext()
-        _init_active_context()
+        initialize_language_context(ACTIVE_CONTEXT)
 
     return ACTIVE_CONTEXT
 
 
-def _init_active_context() -> None:
-    """Initialize the active LanguageContext instance with definitions from the core spec and active plugins."""
-    active_context = get_active_context()
-    active_context.add_definitions_to_context(get_aac_spec())
-    active_context.add_definitions_to_context(get_plugin_definitions())
+def initialize_language_context(language_context: LanguageContext) -> None:
```

Comment:
I would normally agree, but I think the single argument, description, and function name makes it pretty obvious to the reader what's happening. I'm also against the idea of the function returning a new instance of the context, because we already do that with the class constructor, and this function is not intended to replace that constructor.